tools/termux-installation.md

```diff
@@ -35,14 +33,11 @@ apt install openssl wget && wget https://raw.githubusercontent.com/lordbagel42/s
 
 3. Click the APK, and allow it to be installed.
 
-4. Open Termux, and run the following commands.
+4. Open Termux, and run the following command:
 
 ```
-apt update
-termux-change-repo # SELECT MAIN THEN BFSU.
-apt install openssl wget && wget https://raw.githubusercontent.com/lordbagel42/slimevr-termux-installer/main/install.sh && chmod +x install.sh && ./install.sh
+pkg install openssl wget && wget https://raw.githubusercontent.com/lordbagel42/slimevr-termux-installer/main/install.sh && chmod +x install.sh && ./install.sh
 ```
-
```

Comment:
Why remove that newline? Not sure if it changes anything tho.

Reply:
I didn't even notice I removed it lol.

Reply:
Can you add it back (even if it doesn’t change anything, it looks better in the code :p)